plugins/lazy-yt/plugin.rb

```diff
@@ -35,11 +38,15 @@ def to_html
       <div class="onebox lazyYT lazyYT-container"
            data-youtube-id="#{video_id}"
            data-youtube-title="#{escaped_title}"
-           data-width="#{video_width}"
-           data-height="#{video_height}"
+           #{size_restricted ? size_tags.map { |t| "data-#{t}" }.join(' ') : ""}
            data-parameters="#{embed_params}">
         <a href="https://www.youtube.com/watch?v=#{video_id}" target="_blank">
-          <img class="ytp-thumbnail-image" src="#{thumbnail_url}" width="#{video_width}" height="#{video_height}" title="#{escaped_title}">
+          <img class="ytp-thumbnail-image"
+               src="#{thumbnail_url}"
+               #{size_restricted ? size_tags.join(' ') : ""}
+               width="#{video_width}"
+               height="#{video_height}"
```

Comment:
If size_restricted is true, I think this will leave us with duplicate width/height attributes?

Reply:
Oops, forgot to delete those lines. Thanks for the catch!

Fixed in 8a68db4